AlexandraYashina/Homework_8/functional_components/src/pages/UserProfile.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+import User from '../components/User';
+import {connect} from 'react-redux';
+import {fetchUser} from '../actions/userActions';
+
+const UserProfile = props => {
+  const [userInfo, setUserInfo] = useState(['']);
+
+  useEffect(() => {
+    setUserInfo(props.dispatch(fetchUser(props.params.userId)));
```

Comment:
```suggestion
props.dispatch(fetchUser(props.params.userId));
```
вы могли бы написать `setUserInfo(props.user)`, но смысла нет:)